### ISSUE TITLE
WCAG 4.1.2: Ensure aria-hidden elements are not focusable nor contain focusable elements (.thumb.expandable-image[alt=""]).:A11y_VisualStudioCodeServices_Introductory Videos_AI4W

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/vscode-docs#8627]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".

### GitHub Tags:
#A11yTCS; #A11ySev2; #A11yMAS; #DesktopWeb; #ChromiumEdge; #Visual Studio Code Services; #A11yAuto; #AI4W; #BM_VisualStudioCodeServices_Web_July2024; #Win11; #WCAG4.1.2; #Name role value; #aria-hidden-focus;

### Environment Details: 
Application Name:  Visual Studio Code Services
URL: https://code.visualstudio.com/docs/getstarted/introvideos
Microsoft Edge version 138.0.0.0

### Repro Steps:

1. Launch the application using the URL: https://code.visualstudio.com/docs/getstarted/introvideos.
2. "Introductory Videos" screen will appear.
3. Run fast pass and observe the issue.

### Issue: 
Ensure aria-hidden elements are not focusable nor contain focusable elements (aria-hidden-focus - https://accessibilityinsights.io/info-examples/web/aria-hidden-focus)

### Target application: 
Introductory Videos - https://code.visualstudio.com/docs/getstarted/introvideos

### Element path: 
a[href$="basics"] > .thumb.expandable-image[alt=""]

### Snippet: 
<img src="/assets/docs/getstarted/introvideos/getting-started.png" alt="" aria-hidden="true" class="thumb expandable-image" tabindex="0" aria-label="Documentation image. Expandable image, press Enter or Space to view larger and press Escape to close.">

### How to fix: 
Fix all of the following:
  Focusable content should have tabindex="-1" or be removed from the DOM

Environment: Microsoft Edge version 138.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Attachment:

<img width="1365" height="715" alt="Image" src="https://github.com/user-attachments/assets/c595af04-2713-4f82-9c07-03a1882a275c" />

**Comment by rzhao271** [comment]
@Mana-swi I believe Hawk has fixed this issue. Could you take another look?

**Comment by Mana-swi** [comment]
#Closed;#Regressed:08-13-25;#A11yFixed;

Verified the issue in below environment
URL : https://code.visualstudio.com/docs/getstarted/introvideos

The issue is fixed, hence closing the bug. Please find attachment for reference.

Normal view:

<img width="1360" height="694" alt="Image" src="https://github.com/user-attachments/assets/1a652c1a-8cc1-4046-8bf1-6e84035279ff" />

800*600 view:

<img width="1364" height="706" alt="Image" src="https://github.com/user-attachments/assets/169e1ba8-21d9-4148-80d4-695d7dfffaf2" />
